client/landing/stepper/declarative-flow/internals/hooks/use-step-navigation-with-tracking/index.ts

```diff
@@ -69,14 +71,15 @@ export const useStepNavigationWithTracking = ( {
 	return useMemo(
 		() => ( {
 			...( stepNavigation.submit && {
-				submit: ( providedDependencies: ProvidedDependencies = {}, ...params: string[] ) => {
+				// TODO: remove = {}, there is no need to default to {}.
```

Comment:
TODO comment in here.
I don't mind it if we default to `{}`

Reply:
I don't mind either, but this is a decorator that is changing the function signature when it shouldn't.